sqldelight-compiler/src/main/antlr/com/squareup/sqldelight/Sqlite.g4

```diff
@@ -646,17 +653,18 @@ any_name
  | '(' any_name ')'
  ;
 
-// SQLite accepted types
-STRING : S T R I N G;
-INT : I N T;
-LONG : L O N G;
-SHORT : S H O R T;
-FLOAT : F L O A T;
-DOUBLE : D O U B L E;
-BOOLEAN : B O O L E A N;
-BLOB : B L O B;
-ENUM : E N U M;
-CLASS : C L A S S;
+K_JAVA_BOOLEAN: B 'o' 'o' 'l' 'e' 'a' 'n';
```

Comment:
you need to add all these to the `keyword` rule in the grammar so they highlight properly in the IDE and also so you can't type them elsewhere.